poppy/instrument.py

```diff
@@ -436,15 +436,30 @@ def _get_fits_header(self, result, options):
 
         if self.pupilopd is None:
             opdstring = "NONE - perfect telescope! "
+            opdfile = 'None'
+            opdslice = 'None'
         elif isinstance(self.pupilopd, str):
             opdstring = os.path.basename(self.pupilopd)
+            opdfile = self.pupilopd
```

Comment:
For consistency I think this should also be wrapped in `os.path.basename` like the others.